lib/davinci_dtr_test_kit/profiles/communication_request/communication_request_validation.rb

```diff
@@ -0,0 +1,34 @@
+require_relative '../../validation_test'
+
+module DaVinciDTRTestKit
+  class CommunicationRequestValidationTest < Inferno::Test
+    include DaVinciDTRTestKit::ValidationTest
+
+    title 'CommunicationRequest resources returned during previous tests conform to the CRD CommunicationRequest'
+    description %(
+This test verifies resources returned from the first read conform to
+the [CRD CommunicationRequest](http://hl7.org/fhir/us/davinci-crd/StructureDefinition/profile-communicationrequest).
```

Comment:
let's choose a specific version of the CRD IG to point to: https://hl7.org/fhir/us/davinci-crd/STU2/...  Applies to all of these.

Reply:
What should I do about `QuestionnaireResponse`?